frontend/scripts/containers/home/home.container.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import Home from 'react-components/home/home.component';
+import { getURLFromParams, POST_SUBSCRIBE_NEWSLETTER } from 'utils/getURLFromParams';
+import { getSliderContent } from 'actions/home.actions';
+
+// Seens like an overkill to include into redux
+function submitForm(email) {
```

Comment:
this feels more like an redux action than a class method. Eventually we can use it in other places - like the data portal, for example